### PR TITLE
Added SDF nodes to visual shader

### PR DESCRIPTION
**Pull request by Chaosus**
Added 4 new nodes to visual shader(canvas item mode) refer to functions which @reduz added recently

image

@godotengine/documentation please suggest the descriptions

### CODE REVIEW

**Comment by Chaosus** [comment]
Added Raymarch node as @reduz requested (https://www.irccloud.com/pastebin/k37wXtsT/)

image

**Comment by akien-mga** [comment]
I rebased and updated the PR with additional documentation, should be good to merge.

**Review by reduz — APPROVED** [review]

File: editor/plugins/visual_shader_editor_plugin.cpp [hunk 1]
```diff
@@ -3671,7 +3672,17 @@ VisualShaderEditor::VisualShaderEditor() {
 	add_options.push_back(AddOption("FloatUniform", "Scalar", "Variables", "VisualShaderNodeFloatUniform", TTR("Scalar floating-point uniform."), -1, VisualShaderNode::PORT_TYPE_SCALAR));
 	add_options.push_back(AddOption("IntUniform", "Scalar", "Variables", "VisualShaderNodeIntUniform", TTR("Scalar integer uniform."), -1, VisualShaderNode::PORT_TYPE_SCALAR_INT));
 
+	// SDF
+	{
+		add_options.push_back(AddOption("ScreenUVToSDF", "SDF", "", "VisualShaderNodeScreenUVToSDF", "", -1, VisualShaderNode::PORT_TYPE_VECTOR, TYPE_FLAGS_FRAGMENT | TYPE_FLAGS_LIGHT, Shader::MODE_CANVAS_ITEM));
```
Comment (Chaosus) [inline editor/plugins/visual_shader_editor_plugin.cpp:3677]: @akien-mga Here are short descriptions should be.

Comment (akien-mga) [reply editor/plugins/visual_shader_editor_plugin.cpp]: Thanks, should be added now.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**